src/main/java/com/qaacademy/module3/tddgit/Main.java

```diff
@@ -2,6 +2,8 @@
 
 public class Main {
     public static void main(String[] args) {
+
         System.out.println("Hello world!");
+
```

Comment:
Don't need to modifies this class
please rollback these changes, only you need to send your package.
`harold.unittesttdd;`